python-sdk/tests/databases/test_snowflake.py

```diff
@@ -51,7 +55,7 @@ def test_snowflake_run_sql():
 def test_table_exists_raises_exception():
     """Test if table exists in snowflake database"""
     database = SnowflakeDatabase(conn_id=CUSTOM_CONN_ID)
-    table = Table(name="inexistent-table", metadata=Metadata(schema=SCHEMA))
+    table = Table(name="inexistent-table", metadata=Metadata(schema=SNOWFLAKE_SCHEMA))
```

Comment:
Why did we need to change `SCHEMA` to `SNOWFLAKE_SCHEMA` (I see only difference is lower-case / upper-case) -- why though?

Reply:
I think we still need it, since we are `write_pandas()` doesn't give us an option of treating schema and table name differently, everything is an identifier. So for mixed/upper case, we preserver the case of all the identifiers, which will lead the query to form with "astroflow_ci" and that scheme won't be found since it's not there.